HotelBooking/Views/book_reservation_view.py

```diff
@@ -132,8 +132,15 @@ def reserve_room(self):
                     PROMPT_KEY['CUSTOMER'])
             self.start_date = self.prompt_and_get_answer(
                 PROMPT_KEY['START_DATE'])
-            self.duration = int(
-                self.prompt_and_get_answer(PROMPT_KEY['DURATION']))
+            while True:
```

Comment:
For validations on simple inputs that do not require controller/model calls, please valid them using PyInquiry's built-in validation parameter:
```
            {
                'type': 'input',
                'message': "Enter transaction amount in cents",
                'name': 'amount',
                'validate': lambda x: x.isdigit() or "Transaction amount invalid",
            }
```

Reply:
This applies to all other while True validations here.

Reply:
ooo thats so cool